src/modules/renderModule.js

```diff
@@ -1,8 +1,8 @@
 import {
-  getTasksFromStorage, saveTasksToStorage, editTaskDescription, deleteTask,
+  getTasksFromStorage, /* saveTasksToStorage, */ editTaskDescription, deleteTask,
```

Comment:
[OPTIONAL] As I mentioned above :point_up_2: please remove unwanted commented code from your codebase because it's your codebase make messy if you want to check your previous code then check your GitHub commit history :slightly_smiling_face: